### ISSUE TITLE
use original found comic path no matter what

### ISSUE THREAD

**Issue by CuddleBear92** [evilhero/mylar#2130]
I had a bunch of issues where it wouldn't use the original path of the found imported comic.
So I asked a friend for help on the subject and he edited it out so it does this for me.

Originally it always tied the comic path to whatever setting was set in the folder creation/naming format and ignoring the original path all together.

For example, a series path to say: `L:\Comics\DC Comics - Vertigo\Lucifer (V2018) CVID114408` would be saved as: `L:\Comics\Vertigo\Lucifer (V2018)` no matter what. which wasn't up to the standards of my exports.

This file should make it so it uses the original path all along if you have it set to not rename or move files at all. leaving them set to downloaded and not archived too!

EDIT: https://github.com/CuddleBear92/Ubooquity-Themes/tree/master/Settings_%26_Scripts/mylar_custom_code

Note that no actual settings or options was added and prob needs to be added for a full release of this one!

**Comment by evilhero** [comment]
The problem with the gist is that it will attempt to use the imported file path for any non-imported adds (ie. adding a series via another option) and will fail as the import variable is empty. So while it will work for when you do an import, it won't for anything else.

What I'm unsure about here is how to go about things so that it works for users with setups like yours where they wish to maintain the existing paths, while still ensuring things maintain the same status quo for current users so as to not change their setups (ie. they don't want Mylar to do anything but make note that the issues exist in some form).

Am I to understand it right that you just want to set the Series Directory for a series to the scanned in location from an import? In doing this, that means that any new downloads/post-processes will get dumped into that location too.  The reason the Archived status exists is to let users with multiple locations keep one location 'safe' so-to-speak, while allowing new downloads to propagate to a new locaiton.  

This could be accomplished easily enough by just adding an option to the import results page whereby you can select a ```set series directory to imported path``` and in doing so then with a slight modification of the variable being referenced, your gist would work fine for everyone.

**Comment by CuddleBear92** [comment]
We updated both importer and librarysync files atm. please look it over and try to work something like it into the full release. 
Using the path as is does make sense for many use cases as everyone got their own way of sorting files.

Understand that some things won't work perfectly. but I'm guessing it will look for new issues of a known series in that folder.

He also helped me with a cvinfo script that takes the CVID# numbers that ComicRack exported and makes the cvinfo file for perfect matching (I'm guessing it reads that and matches to that automatically if there)
This way I won't get false matches either by doing this as its already matched correctly with the ComicVineScraper.

**Comment by evilhero** [comment]
Your correct about the cvinfo - if Mylar detects the cvinfo in a given directory it assumes everything in that given directory belongs to the cvinfo file. So as long as you don't have multiple volumes in the same directory with one cvinfo file, you'll get the end result you desire.

I'll float your code for importer.py in, but I'll have to add in a configurable option in the import page section so people have a choice as to what they want to do in regards to the paths. 

As for the librarysync, I'm not quite sure about the DB call you did - or to what purpose it's being used for ... it seems that it's to avoid scanning in paths that were already imported previously? If you can explain the reasoning behind it, I don't think I'd have a problem with implementing - just would like to know the use case.

As an aside, you can turn off directory creation from within the config.ini (```create_folders = False```). Directories will then only be created for series when post-processing occurs and files need to be moved/copied into them.

**Comment by CuddleBear92** [comment]
the reasoning for the librarysync stuff is so it doesn't display the series again when scanning/importing the root folders of the series or comics in general.
as it would fill the review import page with the same entries even when imported in the past.
Its so it will only allow to import new series that isn't already in the database, showing only the new stuff.
Issues on an already imported series can just be refreshed after all.


And I did do the create_folders=False too and it still tied the series to the folder rename format which would be wrong.
Seemingly it tied to the wrong path no matter what settings I used by default.
